index.html

```diff
@@ -1,45 +1,75 @@
     <section class="section-presentation">
         <div class="presentation" id="presentation">
             <h1 class="name">I’m Tarik Glad to see you!</h1>
-            <p class="present">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="present">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
             <h4 class="blue text">LET’S CONNECT</h4>
             <div>
                 <ul class="social-media">
-                    <li><a href="https://twitter.com/TarikBouari"><img class="icon" src="images/Vector2.png" alt="twitter"></a></li>
-                    <li><a href="https://www.linkedin.com/in/tarik-bouari-44b7191a6/"><img class="icon" src="images/Linkedin.png" alt="twitter"></a></li>
+                    <li><a href="https://twitter.com/TarikBouari"><img class="icon" src="images/Vector2.png"
+                                alt="twitter"></a></li>
+                    <li><a href="https://www.linkedin.com/in/tarik-bouari-44b7191a6/"><img class="icon"
+                                src="images/Linkedin.png" alt="twitter"></a></li>
                     <li> <a href="#"><img class="icon" src="images/VectorM.png" alt="twitter"></a></li>
-                    <li> <a href="https://github.com/tarikbouari"><img class="icon" src="images/Vector3.png" alt="twitter"></a></li>
+                    <li> <a href="https://github.com/tarikbouari"><img class="icon" src="images/Vector3.png"
+                                alt="twitter"></a></li>
                     <li><a href="#"><img class="icon" src="images/Vector5.png" alt="twitter"></a></li>
-                </ul>    
+                </ul>
             </div>
         </div>
     </section>
 
     <!--Project Section-->
     <section class="projects">
         <div class="project-card">
-             <img src="images/tonic.png" alt="tonic-project" class="img-project">
+            <img src="images/Snapshoot.jpg" alt="tonic-project" class="img-project">
+            <div class="info-project">
+                <h2 class="title">Tonic</h2>
+                <ul class="specific">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="language">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <div><button class="primary-button">See Project</button></div>
+            </div>
+        </div>
```

Comment:
- [ ] Your project cards are very zoomed, please adjust your cards so that they look similar to the one from the Figma design. 
![image](https://user-images.githubusercontent.com/43877689/177634292-f874bccb-ba6b-429c-b95d-0e87881970a4.png)